.github/workflows/blank.yml

```diff
@@ -0,0 +1,96 @@
+name: CI
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      # ovn-kubernetes
+      ENABLE_MULTI_NET: "true"    
+      KIND_INSTALL_INGRESS: "false"
+      KIND_INSTALL_METALLB: "false"
+      KIND_INSTALL_PLUGINS: "true"      
+      KIND_IPV4_SUPPORT: "true"
+      KIND_IPV6_SUPPORT: "false"
+      KUBECONFIG: "/home/runner/ovn.conf"
+      OVN_DISABLE_SNAT_MULTIPLE_GWS: "true" 
+      OVN_GATEWAY_MODE: "shared"
+      OVN_HYBRID_OVERLAY_ENABLE: "false"
+      # web-burner
+      BFD: "false"
+      BRIDGE: "breth0"
+      BURST: "20"
+      ES_SERVER: "https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com"
+      ES_INDEX: "ripsaw-kube-burner"
+      INDEXING: "false"
+      KUBE_BURNER_RELEASE: "1.7.2"
+      LIMITCOUNT: "1" 
+      QPS: "20"
+      SCALE: "1"
+      SRIOV: "false"
+
+    steps:      
+      - uses: actions/checkout@v3
+        with:
+          repository: 'ovn-org/ovn-kubernetes'
```

Comment:
You can checkout just the latest commit with
```
        fetch-depth: 1
```

That should save some time and disk space

Reply:
Thanks @rsevilla87 !
Done, applied [here](https://github.com/redhat-performance/web-burner/blob/main/.github/workflows/blank.yml#L44)